packages/preview/vienna-tech/1.1.0/typst.toml

```diff
@@ -0,0 +1,29 @@
+# for a description of available keys, see https://github.com/typst/packages/?tab=readme-ov-file#package-format
+
+[package]
+name = "vienna-tech"
+version = "1.1.0"
+entrypoint = "src/lib.typ"
+authors = ["Niko Pikall"]
+license = "MIT"
+compiler = "0.13"
```

Comment:
For info the error reported by CI is not true anymore for recent versions of Typst, you can ignore it.